spec.md

```diff
@@ -1076,7 +1081,8 @@ For success responses, the following fields are defined:
 | plan_id | string | The ID of the Service Plan from the catalog that is associated with the Service Instance. |
 | dashboard_url | string | The URL of a web-based management user interface for the Service Instance; we refer to this as a service dashboard. The URL MUST contain enough information for the dashboard to identify the resource being accessed (`9189kdfsk0vfnku` in the example below). Note: a Service Broker that wishes to return `dashboard_url` for a Service Instance MUST return it with the initial response to the provision request, even if the service is provisioned asynchronously. |
 | parameters | object | Configuration parameters for the Service Instance. |
-
+| context | object | Context information provided on provisioning for the Service Instance. |
```

Comment:
Why should the broker reflect the context object back to the platform? It should know the content anyway. Additionally, it would require that brokers must store the original context object, which makes building stateless brokers even more difficult.

Reply:
@fmui indeed this is a good point for stateless brokers and would force them to store data. I will remove it for now and stick to attributes only in this PR.

Reply:
@fmui isn't the case for the parameters as well? it definitely requires the broker to reflect back and I assume it would have to save information as well.